addons/mail/wizard/mail_activity_schedule.py

```diff
@@ -82,7 +85,10 @@ def default_get(self, fields_list):
     @api.depends('res_model')
     def _compute_res_model_id(self):
         for scheduler in self:
-            scheduler.res_model_id = self.env['ir.model']._get_id(scheduler.res_model)
+            if scheduler.res_model:
+                scheduler.res_model_id = self.env['ir.model']._get_id(scheduler.res_model)
+            else:
+                scheduler.res_model_id = False
```

Comment:
Can be simplified with
```suggestion
        self.res_model_id = False
        for scheduler in self.filtered('res_model'):
            scheduler.res_model_id = self.env['ir.model']._get_id(scheduler.res_model)
```